config/locales/de.yml

```
@@ -342,6 +357,22 @@ de:
       user_count_exceeds_plan_limit:
         subject: "%{organization_name} hat die Höchstzahl der %{business_plan_name} Nutzer seines %{users_limit} Plans überschritten "
         text: "%{organization_name} hat die zahl der gebuchten Contributors überschritten. Technisch ist das kein problem. Sprich mit der Kund:in doch demnächst über eine Aufstockung des Plans."
+    whats_app:
+      welcome_message: "Vielen Dank für Ihre Anmeldung bei %{project_name}. Unsere Dialog-Recherche startet bald. Wir melden uns dann bei Ihnen."
+      request_template:
+        new_request_morning_1: "Guten Morgen %{first_name},  zum Thema %{request_title} haben wir eine neue Frage. Möchten Sie darauf antworten?"
+        new_request_morning_2: "Guten Morgen %{first_name},  zum Thema %{request_title} haben wir eine neue Frage. Möchten Sie darauf antworten?"
+        new_request_morning_3: "Guten Morgen %{first_name},  zum Thema %{request_title} haben wir eine neue Frage. Möchten Sie darauf antworten?"
+        new_request_day_1: "Guten Tag %{first_name}, wir haben eine neue Frage zum Thema %{request_title}. Möchten Sie darauf antworten?"
+        new_request_day_2: "Hallo %{first_name}, ich möchte gern mehr zum Thema %{request_title} erfahren. Möchten Sie auf die heutige Frage antworten?"
+        new_request_day_3: "Guten Tag %{first_name}, ich habe eine neue Frage zu %{request_title}. Möchten Sie antworten?"
+        new_request_evening_1: "Guten Abend %{first_name}, wir haben eine neue Frage zum Thema %{request_title}. Möchten Sie darauf antworten?"
+        new_request_evening_2: "Guten Abend %{first_name}, zum Thema %{request_title} haben wir eine Frage. Antworten Sie uns darauf?"
+        new_request_evening_3: "Hallo %{first_name}, heute Abend möchte ich gern mehr zum Thema %{request_title} erfahren. Würden Sie meine Frage beantworten?"
+        new_request_night_1: "Hallo %{first_name}, heute Nacht ist mir etwas zum Thema %{request_title} eingefallen. Antworten Sie mir?"
+        new_request_night_2: "Hallo %{first_name}, ich weiß, es ist spät. Aber ich möchte gern mehr zum Thema %{request_title} erfahren. Möchten Sie auf meine Frage antworten?"
+        new_request_night_3: "Hallo %{first_name}, zum Thema %{request_title} möchte ich heute Nacht gern mehr erfahren. Beantworten Sie mir eine kurze Frage?"
```

```suggestion
        new_request_night_3: "Hallo %{first_name}, zum Thema %{request_title} möchte ich heute nacht gern mehr erfahren. Beantwortest du mir eine kurze Frage?"
```
All of these use formal German. Since we address our contributors by `first_name` I guess we should use informal German. This would need to be changed for all the above :point_up: